trusted/trts_sys/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "mc-sgx-trts-sys"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+default = []
+hw = []
+
+[dependencies]
+mc-sgx-trts-sys-types = { path="../trts_sys_types" }
```

Nit: Space around `=`
```suggestion
mc-sgx-trts-sys-types = { path = "../trts_sys_types" }
```